kazel/kazel.go

```diff
@@ -774,13 +824,13 @@ func writeFile(path string, f *bzl.File, exists, dryRun bool) (bool, error) {
 
 func context() *build.Context {
 	return &build.Context{
-		GOARCH:      "amd64",
-		GOOS:        "linux",
+		GOARCH:      build.Default.GOARCH,
```

Comment:
What is this value when I run on mac?

Reply:
I should rip this out, I don't think we need it anymore

Reply:
yeah, confirmed this is basically unused, since we don't manage go rules anymore. I'll rip it out in a follow-up PR.